gradle-sls-packaging/build.gradle

```diff
@@ -38,6 +37,18 @@ pluginBundle {
             displayName = 'Creates SLS pod distributions'
         }
     }
+
+    // Fix for publishPlugins not working with dependency recommenders (nebula, in our case).
+    // Gradle suggests the 'withDependencies' approach in the following workaround:
+    // http://plugins.gradle.org/help/plugin/missing-dependency-version
```

Comment:
@dansanduleac, do you know why it doesn't fix it. From the readme, the plugin is created exactly for this purpose.

Reply:
That plugin fixes the [POMs](https://github.com/nebula-plugins/nebula-publishing-plugin/blob/0a244ec3fba3c279553c437f5064b7bf98764486/src/main/groovy/nebula/plugin/publishing/maven/MavenResolvedDependenciesPlugin.groovy#L38) of maven publications. This is not a maven publication, but a bespoke code path belonging to `com.gradle.plugin-publish` (which publishes plugins to the Plugin Portal - not sure why it needs the deps of the whole plugin bundle, though!).
Other people have been confused as to why that plugin does nothing here: https://discuss.gradle.org/t/pluginpublish-task-not-respecting-nebula-maven-publish-rules-to-publish-a-resolved-pom/22832